pebble/pathapp/src/pathapp.c

```diff
@@ -0,0 +1,285 @@
+#include "pebble.h"
```

Comment:
Can you add unit tests for this and every app related file? I notice there are none!